src/components/book-card.astro

```diff
@@ -32,7 +32,9 @@ if (data.number) {
 const numberClasses = twMerge(
   "book-number absolute top-[-1.75rem] right-4 sm:right-2 text-[2rem]/none text-white rounded-full bg-black inline-flex flex-col items-center justify-center w-[1.75em] h-[1.75em]",
   collection === "briefs" &&
-    "rounded-lg text-[1.75rem]/none w-auto h-auto py-1 px-2"
+    "rounded-lg text-[1.75rem]/none w-auto h-auto py-1 px-2",
+  collection === "lost" && 
+    "rounded-none transform rotate-45"
```

Comment:
i dont like the un-doing of the default `rounded` class, but that seemed the least invasive.

Reply:
This seems fine, may be an opportunity to refactor so the main, briefs, and lost badges are e.g. separate components, but that doesn't need to be a ship blocker here.